ontology.ttl

```diff
@@ -1,469 +1,1370 @@
+  rdfs:comment "A named collection of books that are produced over a period of time" .
+
+nva:Series a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Series" ;
+  rdfs:subClassOf nva:BookSeries ;
+  rdfs:comment "A known and documented BookSeries" .
+
+nva:UnconfirmedSeries a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "UnconfirmedSeries" ;
+  rdfs:subClassOf nva:BookSeries ;
+  rdfs:comment "A postulated BookSeries that has not been verified as existing" .
+
+nva:PublishingHouse a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "PublishingHouse" ;
+  rdfs:subClassOf nva:Organization ;
+  rdfs:comment "An organization responsible for the making available of a resource" .
+
+nva:Publisher a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Publisher" ;
+  rdfs:subClassOf nva:PublishingHouse ;
+  rdfs:comment "A known organization responsible for the making available of a resource" .
+
+nva:UnconfirmedPublisher a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "UnconfirmedPublisher" ;
+  rdfs:subClassOf nva:PublishingHouse ;
+  rdfs:comment "A postulated, but unverified, organization responsible for the making available of a resource" .
+
+nva:Pages a rdfs:Class ; # Should probably be "Extent" to include "length (temporal), size (inches), size (kB), etc.
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Pages" ;
+  rdfs:comment "A description of the extent of a resource" .
+
+nva:MonographPages a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "MonographPages" ;
+  rdfs:subClassOf nva:Pages ;
+  rdfs:comment "An extent associated with monographs, including the introduction, main body and illustrations" .
+
+nva:Range a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Range" ;
+  rdfs:subClassOf nva:Pages ;
+  rdfs:comment "An extent in terms of an interval with a beginning and an end" .
+
+nva:AdditionalIdentifier a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "AdditionalIdentifier" ;
+  rdfs:comment "An identifier that is supplementary to standard identifiers used in NVA, often retained from legacy systems" .
+
+# Subjects
+
+nva:Concept a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Concept" ;
+  rdfs:comment "An shared idea or mental thing" .
 
 # Files
 
-:FileSet a rdfs:Class ;
-  rdfs:comment "The set of Files associated with a Publication" .
-
-:File a rdfs:Class ;
-  rdfs:comment "A File associated with a Publication" .
+nva:FileSet a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "FileSet" ;
+  rdfs:comment "A collection of files" .
+
+nva:File a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "File" ;
+  rdfs:comment "A digital document" .
+
+nva:License a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "License" ;
+  rdfs:comment "A rights document defining what is permitted and/or forbidden for whom" .
+
+# Projects
+
+nva:Project a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Project" ;
+  rdfs:comment "A time-limited effort towards a common goal" .
```

Comment:
Could it be a freely defined project or does it have to be registered somewhere?